frontend/src/employee-frontend/components/unit/unit-reservations/ChildDay.tsx

```diff
@@ -126,6 +137,24 @@ export default React.memo(function ChildDay({
     serviceTimeOfDay
   )
 
+  const unitStartAfterReservationStart =
+    dailyData.attendance === null &&
+    intermittent &&
+    reservation !== null &&
+    (day.time === null ||
+      day.isHoliday ||
+      (reservation.type === 'TIMES' &&
+        day.time.start > reservation.startTime)) &&
+    !inOtherUnit
```

Comment:
Here it seems that the resulting boolean is not only about whether unit start is after reservation start or not. Maybe naming to something such as `showUnitStartAfterReservationStartWarning` or possibly splitting it into two booleans which would be combined later with && would be more accurate.

Reply:
Renamed to `requiresBackupCareForReservationStart/End`